src/auth/AuthManager.ts

```diff
@@ -34,7 +34,8 @@ export class AuthManager {
 
     this.userName = loginParams.username
 
-    const { isLoggedIn, loginForm } = await this.checkSession()
+    let { isLoggedIn, loginForm } = await this.checkSession()
```

Comment:
why use let instead of const? As these variables ( isLoggedIn and loginForm ) are not being reassigned, its better to use const.